src/mobile/src/ui/views/wallet/Send.js

```diff
@@ -217,7 +217,11 @@ export class Send extends Component {
         if (seedIndex !== newProps.seedIndex) {
             this.resetMaxPressed();
         }
-        if (address.length === 0 && newProps.address.length === 90) {
+        if (this.props.deepLinking && !this.props.deepLinkRequestActive && newProps.deepLinkRequestActive) {
+            this.completeActiveDeepLink();
+            await Clipboard.setString('');
```

Comment:
I believe this should be `Clipboard.setString(' ');`

Reply:
Could you suggest the change on Github?